platform/autoupdate.cli/src/org/netbeans/modules/autoupdate/cli/ModuleOptions.java

```diff
@@ -482,14 +482,22 @@ public CLIInternalHandle(String displayName, Env env) {
         public CLIProgressUIWorker(Env env) {
             this.env = env;
         }
+
         @Override
         public void processProgressEvent(ProgressEvent event) {
-            env.getOutputStream().println(event.getMessage());
+            printEvent(event);
         }
+
         @Override
         public void processSelectedProgressEvent(ProgressEvent event) {
-            env.getOutputStream().println(event.getMessage());
+            printEvent(event);
+        }
+
+        private void printEvent(ProgressEvent event) {
+            final String msg = event.getMessage();
+            if (msg != null && msg.length() > 0) {
```

Comment:
Without this check the output was full of `null`s:
![obrazek](https://user-images.githubusercontent.com/26887752/98004171-e4538480-1def-11eb-9e01-58c93111d42a.png)